paddle/fluid/operators/jit/gen/jitcode.h

```diff
@@ -98,7 +98,7 @@ class JitCode : public GenBase, public Xbyak::CodeGenerator {
     ret();
   }
   void L(const char* label) { Xbyak::CodeGenerator::L(label); }
-  void L(const Xbyak::Label& label) { Xbyak::CodeGenerator::L(label); }
```

Comment:
I'm wondering why you've removed `const` ?  Does this function modify `label` ?

Reply:
Because I've changed Xbyak library version from  v5.661 to v5.81  it turned out that one of the functions has changed from `void L (const Label & label)` to void `L (Label & label)`, so we also have to remove const so that there is no error because there are actually operations on the Label class.